ci/build_windows.py

```diff
@@ -193,14 +238,19 @@ def windows_package(args):
         for dll in dlls:
             logging.info("packing dll: %s", dll)
             shutil.copy(dll, pkgdir_lib)
+
+        if pkgdir_lib.find('gpu') != -1:
+            add_path("C:\\jenkins_slave\\workspace\\ut-python-gpu\\windows_package\\lib")
```

Comment:
Updating the path should be done in the testing phase - in ci/windows/test_py3_gpu.ps1 so the tests can find all the DLLs.

Reply:
That's right, forgot to remove that. Will add that to the testing phase once we got the log from ad95025